src/platform_impl/linux/mod.rs

```diff
@@ -442,7 +442,7 @@ impl Window {
             #[cfg(feature = "x11")]
             Window::X(ref w) => w.request_user_attention(_request_type),
             #[cfg(feature = "wayland")]
-            _ => (),
+            Window::Wayland(ref w) => w.request_user_attention(_request_type),
```

Comment:
Can remove the `_` now that there's definitely always going to be a usage for `_request_type`.